ruby/README.md

```diff
@@ -81,10 +81,10 @@ Or install it yourself as:
 
 ## API
 
-### Miscreant::AES::SIV
+### Miscreant::Internals::SIV
```

Comment:
I'm going to try to add a `Miscreant::AEAD` API which is what should get documented

Reply:
Sounds good!

Reply:
Updated the PR to use future-facing `Miscreant::AEAD` namespace